### ISSUE TITLE
x/pkgsite: documentation feedback - tracking issue

### ISSUE THREAD

**Issue by julieqiu** [golang/go#37819]
This is an umbrella issue to keep track feedback specific to documentation rendering for pkg.go.dev.

Issues currently open on this topic are:

- [x] #37816: insert space after Example
- [x] #36808: add ability to link to package doc subheadings
- [ ] #36865: examples missing playground links
- [ ] #37232: support different GOOS/GOARCH values when displaying package documentation
- [ ] #37520: example links do not open examples
- [ ] #37291: example code should be displayed as package main when rendering
- [x] #38056: automatically link RFCs in package documentation
- [ ] #38073: display table of contents in command documentation
- [x] #37863: missing source files listing
- [x] #38205: broken source links in autogenerated API docs
- [x] #38429: hyperlink does not inherit version
- [ ] #38575: anchor links to package-scoped constants and variables appear off-screen
- [ ] #37102: show the package version that introduced a change in documentation
- [x] #38613: type definition links are missing in v2 packages
- [ ] #38699: const block comment not rendered in some cases
- [x] #39270: documentation misparsed as an anchor
- [x] #39529: missing documentation headings/anchors
- [ ] #39907: leave a sample of values when trimming large string literals and composite literals
- [ ] #39559, #39981: new source meta tag + features blocked by that implementation

At the moment, our priority is to work towards open sourcing (#36747) and to address  [design feedback](https://github.com/golang/go/issues?q=is%3Aopen+label%3Ago.dev+label%3AUX) with our UX team. This issue is currently used to track issues that users are running into, to make sure we address them later on.

/cc @dmitshur @jba @spf13 @fflewddur @Joanne881107

**Comment by benhoyt** [comment]
Can we add https://github.com/golang/go/issues/37863 to this list -- the missing "Package Files" section? That's certainly a lack I've noticed a few times.

**Comment by julieqiu** [comment]
Yup - it's been added!

**Comment by julieqiu** [comment]
Closing in favor of milestone pkgsite/dochtml: https://github.com/golang/go/milestone/165.
